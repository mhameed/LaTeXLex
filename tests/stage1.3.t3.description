author: ehc
date: 09/03/03
description: test to check that braces specifically allowed not to match are allowed not to without affecting the other functionality. If you don't believe that this test is required then remove the lines with comment /*These braces are specifically allowed not to match*/ from stage 1.3 and wathch it fail ;)
