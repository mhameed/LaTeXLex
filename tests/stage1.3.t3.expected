Here is some text which includes a close brace that should not be matched \} and some bold text that includes an open brace that should not be matched \{
